src/docbuilder/rustwide_builder.rs

```diff
@@ -93,15 +94,27 @@ impl RustwideBuilder {
             .map(|t| Cow::Owned(t))
             .unwrap_or_else(|_| Cow::Borrowed("nightly"));
 
+        let cpu_limit = std::env::var("DOCS_RS_BUILD_CPU_LIMIT")
```

Comment:
I think we should consider setting a limit even if the environment variable isn't set. #cores - 1 sounds reasonable, is there a way to detect the number of CPU cores using the stdlib or would we need a crate?

Reply:
Looks like there's a crate for this: https://docs.rs/num_cpus/

Reply:
Hmm I'd prefer not to set a limit. If we end up splitting the build server and the web server we'll want the web server to use all cores.